web/styles/design-system/tokens.css

```
@@ -0,0 +1,324 @@
+  --container-product: 80rem;  /* dense product chrome */
+}
+
+/* ============================================================================
+   Semantic element styles — drop these in any page that imports the system.
+   ============================================================================ */
+
+html {
+  font-size: 100%;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.11.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)